pkgs/applications/networking/ipfs/default.nix

```diff
@@ -2,21 +2,26 @@
 
 buildGoModule rec {
   pname = "ipfs";
-  version = "0.4.23";
+  version = "0.5.0";
   rev = "v${version}";
 
+  buildPhase = ''
+    sed -i 'd#\$(d)/preload.sh > $@#' plugin/loader/Rules.mk
```

Comment:
What does removing this file fix?

Reply:
Without removing this, the build process fails, claiming that the preload.sh file doesn't exist (despite the fact that inserting a stat or ls just before shows it perfectly). After several hours of bashing my head against this very strange issue, I tried removing it, and found that it compiled just fine without it.

Reply:
I patched the shebang instead. Than the script works just fine.